### DISCUSSION TITLE
[External] Adding album cover art component

### DISCUSSION THREAD

**Discussion by SB2DD** [gickowtf/pixoo-homeassistant#42] [Ideas]
From [https://community.home-assistant.io/t/version-1-7-2-is-online-divoom-pixoo-64-integration/636885/11?](https://community.home-assistant.io/t/version-1-7-2-is-online-divoom-pixoo-64-integration/636885/11)

> Hey [@gickowtf](https://community.home-assistant.io/u/gickowtf) ,
I appreciate your amazing work! I have a solution that can automatically show the album cover art of whatever the media player is playing. Maybe you would like to try it or even add this feature to your code. [GitHub - idodov/pixoo64-media-album-art: Showing what is playing (Album Cover... 9](https://github.com/idodov/pixoo64-media-album-art)

**Comment by SB2DD** [comment]
I personally think it would be nice to implement a "web image" component in component pages. This could be quite easy to do.

I'm saying this since the original integration gets the album picture by getting the picture from (HA_URL)(/)(The entity_picture attribute of the media player entity). Therefore, the integration would only have to add support to web images, and that would be done.

**Comment by gickowtf** [comment]
hey yeah sure that's a good idea is just the question how we do that with the image size always scale to 64x64??

**Reply by SB2DD** [reply]
I was actually working on it today. I was using a "resize" tag, where you can put the desired final size of the image. I'd like to introduce this to the regular image component too.

**Reply by gickowtf** [reply]
yeah that's sounds good

**Reply by SB2DD** [reply]
Added in #44

**Reply by gickowtf** [reply]
tomorrow ill test it with Apple Music covers
